### ISSUE TITLE
The pin example grows over time

### ISSUE THREAD

**Issue by halt-hammerzeit** [michaelvillar/dynamics.js#20]
Open this example
http://dynamicsjs.com/examples/pin.html

And leave it for ~10 minutes.
In my case (Chrome, Windows, DevTools open) the `svg` slowly grows bigger than the `.pin`.
Maybe that's because the previous `transform3d` is not identity matrix when the new `transform3d` is applied.

**Comment by michaelvillar** [comment]
Yes definitely a problem. One solution would be to have an optional `from` parameter to setup attributes at their initial position. I'm not sure I'll work on that soon.

**Comment by dalisoft** [comment]
This issue comes from performance.now and rAF timing. With date.now this issue will be fixed
